client/src/app/slides/motions/motion/motions-motion-slide.component.html

```diff
@@ -1,4 +1,38 @@
 <div *ngIf="data">
-    Motion Slide
-    <h1>{{ data.data.title }}</h1>
+    <div id="sidebox" *ngIf="data.data.show_meta_box">
+        <!-- State -->
+        <h3 translate>State</h3>
+        {{ data.data.state }}
+        <span *ngIf="data.data.state_extension">{{ data.data.state_extension }}</span>
+
+        <!-- Recommendation -->
+        <div *ngIf="data.data.recommendation">
+            <h3 translate>Recommendation</h3> <!-- TODO: use config 'motions_recommendations_by'-->
+            {{ data.data.recommendation }}
```

Comment:
motions with recommendation failed with server error #4190

Reply:
This might be possible, I didn't check the code there. If you can repair it, go on! We (as the client devs) know, what data we need, so we must puzzle it together from the given data on the server.

Reply:
Fixed